readme.txt

```diff
@@ -12,101 +12,90 @@ The easiest & fastest Contact Form on WordPress. Multiple templates, drag-&-drop
+* **One Page Submission:** Form submission occurs without a page refresh using AJAX, making submissions instant
+* **Field Customization Options:** Customize the form fields available with tons of options to best fit your needs.
+* **Form Redirection:** Upon form submission easily redirect your users to any page of your choice
+* **Form Submission Notification:** Use our built-in email notifications to never miss a beat when your users submit a form on your website 
+* **Manage Form Submissions:** Manage the submission of forms to collect data without the hassle of backend coding
+* **Import or Export Forms:** Use the quick form exporting and importing option that lets you start from where you left off or save your data offline
+* **Mobile-friendly forms:** weForms is mobile-friendly and responsive, making it easy for your users to submit forms via computer, mobile phone, or tablet
+* **Mail support:** Get complete mail support feature in the form of customize mails and HTML email template
+* **Email submission data:** Connect your forms to several of our integrated email services to get form submission data in emails
+* **Anti-spam & reCaptcha:** Get rid of spam entries with reCaptcha or through our other integrated partners, like Human Presence to eliminate spam messages and registrations with WordPress forms
+* **Slack Integrations:** Get notified through your Slack account whenever any user submits a form using weForms
+* **Column Field:** Build your form the way you want by adding any field into our column field. Resize them the way you need them to create a dynamic form for your website
+
+= KEY ATTRIBUTES - WEFORMS STANDS OUT FROM THE CROWD! =
 
 = DRAG & DROP FORM BUILDER WITH LIVE FORM PREVIEW  =
-If you are not using weForms, then you must be using a slower or a buggy form builder in your WordPress. With its smooth drag and drop form builder, create any type of forms in minutes. You won’t be required to know a single line of coding to create some really powerful forms with the help of its intuitive drag & drop form building features. Just simply drag & drop the fields you want to add to your form from the right panel of the available form fields. Instantly live preview form that you have created in the frontend and make any changes you want right away. Use various types of forms in any department of your company going beyond creating just a simple contact or order form making the workflow totally seamless.
+With weForms smooth drag and drop form builder, create any type of form you need within minutes. You aren’t required to know a single line of code to create a powerful form with the help of our intuitive drag & drop form building feature. Simply drag & drop the fields you want to add from the panel of the available form fields. Instantly live preview the form that you have created in the frontend. 
```

Comment:
Starting a sentence with the word "With" is a little awkward.  See https://www.dailywritingtips.com/avoid-beginning-a-sentence-with-with/